pcmdi_metrics/sea_ice/README.md

```diff
@@ -28,11 +30,17 @@ sea_ice_driver.py -p parameter_file.py --msyear 1991 --meyear 2020
 
 The driver produces a JSON file containing mean square error metrics for all input models and realizations relative to the reference data set. It also produces a bar chart displaying these metrics.
 
+### Sectors
+
+The metrics results are provided for eight different geographic regions. In the Northern Hemisphere there are the Arctic, North Pacific, North Atlantic, and Central Arctic regions. In the Southern Hemisphere there are the Antarctic, South Pacific, South Atlantic, and Indian Ocean regions. The region definitions can be found in Ivanova (2016).
+
 ## Parameters
 
+A [demo parameter file](https://github.com/PCMDI/pcmdi_metrics/blob/405_sic_ao/pcmdi_metrics/sea_ice/param/parameter_file.py) is provided in the sea ice code.  
+
 * **case_id**: Save JSON and figure files into this subdirectory so that results from multiple tests can be readily organized.
 * **test_data_set**: List of model names.
-* **realization**: List of realizations.
+* **realization**: List of realizations or "*" to use all realizations.
 * **test_data_path**: File path to directory containing model/test data.
 * **filename_template**: File name template for test data, e.g., "CMIP5.historical.%(model_version).r1i1p1.mon.%(variable).19810-200512.AC.v2019022512.nc" where "model_version" and "variable" will be analyzed for each of the entries in test_data_set and vars.
```

Comment:
Did we want to use a CMIP6 template, which may reorder this information a little?

Reply:
@durack1 Can you explain? I'm not familiar with this template and how it affects the parameters.

Reply:
@acordonez no problem. So the folders/files templates are as follows:
```python
CMIP6 path: CMIP6/CMIP/CSIRO/ACCESS-ESM1-5/historical/r1i1p1f1/Amon/tas/gn/v20191115/
CMIP6 file: tas_Amon_ACCESS-ESM1-5_historical_r1i1p1f1_gn_185001-201412.nc
CMIP5 path: cmip5/output1/CSIRO-BOM/ACCESS1-3/historical/mon/atmos/Amon/r1i1p1/v20130325/tas/
CMIP5 file: tas_Amon_ACCESS1-3_historical_r1i1p1_185001-200512.nc
```
For CMIP6 we have template examples [here](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/f70febdb3a61b24b2c6c685a698c5dda6e5ee594/CMIP6_DRS.json#L3-L6)
```python
    "DRS":{
        "directory_path_example":"CMIP6/CMIP/MOHC/HadGEM3-GC31-MM/historical/r1i1p1f3/Amon/tas/gn/v20191207/",
        "directory_path_sub_experiment_example":"CMIP6/DCPP/MOHC/HadGEM3-GC31-MM/dcppA-hindcast/s1960-r1i1p1f2/Amon/tas/gn/v20200417/",
        "directory_path_template":"<mip_era>/<activity_id>/<institution_id>/<source_id>/<experiment_id>/<member_id>/<table_id>/<variable_id>/<grid_label>/<version>",
        "filename_example":"tas_Amon_HadGEM3-GC31-MM_historical_r1i1p1f3_gn_185001-186912.nc",
...
```

Reply:
Do you mean replacing the example template string with a CMIP6 one? We can, the PMP will work with any arbitrary filename template if it points to an existing file and uses the allowed %() modifiers.

Reply:
Sorry, I wasn't clear. In the demo, we're using CMIP6 data, but this template looks like a CMIP5 template to me, so a switchout to the relevant template would be useful

Reply:
@durack1 I've update the example